src/account_multisig.ts

```diff
@@ -21,13 +21,56 @@ export const MULTISIG_CHANGE_METHODS = ['add_request', 'add_request_and_confirm'
 export const MULTISIG_VIEW_METHODS = ['get_request_nonce', 'list_request_ids'];
 export const MULTISIG_CONFIRM_METHODS = ['confirm'];
 
+/********************************
+This method can be used to mod a near instance .account helper to return AccountMultisig if the account has multisig deployed
+It takes an options object where you can provide callbacks for:
+- sendCode: how to send the 2FA code in case you don't use NEAR Contract Helper
+- getCode: how to get code from user (use this to provide custom UI/UX for prompt of 2FA code)
+- onResult: the tx result after it's been confirmed by NEAR Contract Helper
+********************************/
+export const modIfMultisig = (near, options) => {
```

Comment:
I think it's best if it is just default behavior of `near.account`. In a sense that `await near.account(accountId)` just figures out what is proper instance for given `accountId` and returns it.

However this means that stuff like callback to get code will need to be passed through main config when calling `connect`

Reply:
I'm trying to avoid a backwards compat issue.

Reply:
@mattlockyer should we add option to config? `near.connect({...config, 2fa: true })` or something like that?